translate/synth.metadata

```diff
@@ -1,19 +1,19 @@
 {
-  "updateTime": "2019-10-05T12:42:06.391804Z",
+  "updateTime": "2019-10-18T22:49:41.466785Z",
   "sources": [
     {
       "generator": {
         "name": "artman",
-        "version": "0.38.0",
-        "dockerImage": "googleapis/artman@sha256:0d2f8d429110aeb8d82df6550ef4ede59d40df9062d260a1580fce688b0512bf"
+        "version": "0.40.0",
```

Comment:
Will this need to be a major version bump?

Reply:
or does versioning happen elsewhere?

Reply:
I'll cut a release PR after this one to do the version bump in the `setup.py`. :+1:  The version in `synth.metadata` is the library generator version.

Reply:
Gotcha.